Formula/rex.rb

```diff
@@ -205,10 +227,18 @@ class Rex < Formula
     sha256 "771f7d02abd1ded94d9e37d3f66e795c8d2026d04defbeb5b679ca058116bbf3"
   end
 
+  resource "inc::latest" do
+    url "http://search.cpan.org/CPAN/authors/id/D/DA/DAGOLDEN/inc-latest-0.500.tar.gz"
+    sha256 "daa905f363c6a748deb7c408473870563fcac79b9e3e95b26e130a4a8dc3c611"
+  end
+
   def install
     ENV.prepend_create_path "PERL5LIB", libexec/"lib/perl5"
     ENV.prepend_path "PERL5LIB", libexec/"lib"
 
+    # Spare Perl v5.23+ users a long-winded gripe from JSON::XS module's author.
```

Comment:
Yeah, rex. I don't think there's any hope of getting the module author to change his opinions, but I do think we can reasonably expect rex to say sure we'll change the dep to something supported, or no we'd rather not change the dep despite the downstream consequences. Either way, that will provide a rational basis for either keeping or removing rex.

But perl will be upgraded regardless, and, no, we do not need to wait for them to respond.